src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -16,6 +16,8 @@ public class KNNConstants {
     public static final String KNN_METHOD = "method";
     public static final String NAME = "name";
     public static final String PARAMETERS = "parameters";
+    public static final String MAX_CONNECTIONS = "max_connections";
```

Comment:
These arent going in the API right?

Reply:
They are part of API, I didn't make any change here. This is the `m` value right ?